lib/spacesuit/cors_middleware.ex

```diff
@@ -139,7 +139,7 @@ defmodule Spacesuit.CorsMiddleware do
   end
 
   defp valid_control_headers?(headers) do
-    MapSet.subset?(headers, access_control_request_headers())
+    MapSet.size(access_control_request_headers()) == 0 || MapSet.subset?(headers, access_control_request_headers())
```

Comment:
sorry I'm looking at this after merged, but an optimization for a future patch; in case where there are configured headers, this calls [same logic twice](https://github.com/relistan/spacesuit/blob/master/lib/spacesuit/cors_middleware.ex#L240). I.e.,
```
defp access_control_request_headers do
    (cors()[:access_control_request_headers] || [])
    |> Enum.map(&String.downcase/1)
    |> Enum.into(MapSet.new)
  end
```